recipes/compyle/LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+Unless otherwise specified by LICENSE.txt files in individual
```

Comment:
This file doesn't need to exist as it's included in the tarball

Reply:
Yeah, but it was giving an error during the build process that License was missing, that is why I though of including it as well.